Makefile

```diff
@@ -81,6 +81,7 @@ GO_BUILD       = env CGO_ENABLED=0 GOOS=$(PLATFORM) GOARCH=$(ARCH) \
                   -ldflags '-s -w -X $(SC_PKG)/pkg.VERSION=$(VERSION) $(BUILD_LDFLAGS)'
 
 BASE_PATH      = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
+ORIG_GOPATH   ?= $(shell go env GOPATH)
```

Comment:
This is necessary because we are rewriting the GOPATH env var below to point to a directory local to our repo, which is not where we really want to install svcat.